src/theory/quantifiers/cegqi/ceg_instantiator.cpp

```diff
@@ -125,6 +125,26 @@ std::ostream& operator<<(std::ostream& os, CegHandledStatus status)
   return os;
 }
 
+/**
+ * Return the result of multiplying constant integer or real nodes c1 and c2.
+ * The returned type is real if either have type real.
+ */
+Node multConstants(const Node& c1, const Node& c2)
```

Comment:
Maybe this could be a more general utility?

Reply:
Sure, I can move this to `arith_utilities.h`